pkg/config/config.go

```diff
@@ -465,6 +466,7 @@ func NewConfig(editors ...ConfigEditor) config {
 			},
 			Seed: seed{
 				Enabled: true,
+				Path:    []string{"./seed.sql"},
```

Comment:
```suggestion
```

not necessary as `templates/config.toml` will fill it up if left empty

Reply:
There is cases in our tests where we don't call the `Load` config but just use `NewConfig()`, so this values isn't set by the template and it end up causing tests failure. I could try to refactor the tests to make them call `Load` just didn't looked like it was the best option.